### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by SwatiSima** [github/india#376] [Students]
I have watched industry session videos without fast forwarding or reloading but still it shows incomplete. What should I do now?

**Comment by ghost** [comment]
If there is any buffering or if you pause the video or if you skip the video for few seconds or if you reload the video then it will show incomplete.
example: if the video is for 33.55 min, it should be completed in exactly 33.55 min only.

**Reply by SwatiSima** [reply]
I know about it and I tried to watch so many times still it shows incomplete

**Reply by TheSuranaverse** [reply]
@SwatiSima 
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
